_inc/client/plans/single-product-backup.jsx

```diff
@@ -61,24 +63,54 @@ export function SingleProductBackup( { products, upgradeLinks } ) {
 	);
 }
 
-function SingleProductBackupHeader( { currencySymbol, backupPlanPrices } ) {
+function SingleProductBackupHeader( { currencySymbol, currencyCode, backupPlanPrices } ) {
 	return (
 		<div className="single-product-backup__header-container">
 			<h3 className="single-product-backup__header-title">{ __( 'Jetpack Backup' ) }</h3>
-			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
+			<PlanPriceDisplay
+				currencySymbol={ currencySymbol }
```

Comment:
If we use `formatCurrency` function from `@automattic/format-currency` we won't need the `currencySymbol` - the `currencyCode` would suffice.